upload.sh

```diff
@@ -106,6 +124,11 @@ if [ "$TRAVIS_COMMIT" != "$tag_sha" ] ; then
 
 fi # if [ "$TRAVIS_COMMIT" != "$tag_sha" ]
 
+if [ x"$release_url" = "x" ] ; then
```

Comment:
You could've used `-z` here, too, like in the other comparisons, to keep it consistent.

Reply:
True. I can redo that and resubmit if you prefer

Reply:
I don't think it really matters, you can leave it as it is.